test/Transforms/Passes/tuple-to-struct.mlir

```diff
@@ -0,0 +1,99 @@
+// RUN: p4mlir-opt --p4hir-tuple-to-struct %s | FileCheck %s
+
+!b32i = !p4hir.bit<32>
+!i32i = !p4hir.int<32>
+!_tupleToStruct = !p4hir.struct<"_tupleToStruct", element_0: !b32i, element_1: !i32i>
+
+// CHECK-LABEL: module
+// CHECK-LABEL: p4hir.func @bid
+// CHECK-SAME: (%arg0: !_tupleToStruct) -> !_tupleToStruct
```

Comment:
You really need to match the resulting type as type name alias in general is meaningless. So you'd do:

`![[tupleToStructType:.*]] = !p4hir.struct<"_tupleToStruct", element_0: !b32i, element_1: !i32i>`

And then match using `[[tupleToStructType]]` variable name.